tests/test_session.py

```diff
@@ -202,3 +203,11 @@ def test_get_fluent_mode(new_mesh_session):
     assert session.fluent_connection.get_current_fluent_mode() == "meshing"
     session = session.switch_to_solver()
     assert session.fluent_connection.get_current_fluent_mode() == "solver"
+
+
+def test_unsuccessful_fluent_connection():
+    # start-timeout is intentionally provided to be 2s for the connection to fail
+    logging.disable(logging.CRITICAL)
```

Comment:
why this is needed?

Reply:
@mkundu1, it was logging and error of time-out.

Reply:
this was given just so that line is not logged during testing.

Reply:
Ok, probably a good idea to enable it after the test

Reply:
@mkundu1, ohk, thank you Mainak. This has been applied.

Reply:
@prmukherj with the new fix in place, we no longer need this.

Reply:
This test has been skipped altogether now. We can run this only locally.